autobump-core/src/main/java/com/github/autobump/core/model/Dependency.java

```diff
@@ -1,12 +1,12 @@
 package com.github.autobump.core.model;
 
-import lombok.Builder;
-import lombok.Value;
+import lombok.Data;
+import lombok.experimental.SuperBuilder;
 
-@Value
-@Builder
+@Data
```

Comment:
Not blocking for this PR but we should try to avoid having a mutable dependency object in the core if possible.

Reply:
we needed the data anotation for inheritance to the mavenDependency